src/PortToTripleSlash/src/libraries/AllTypesVisitor.cs

```diff
@@ -13,7 +13,13 @@ internal class AllTypesVisitor : SymbolVisitor
         public readonly List<ISymbol> AllTypesSymbols = new();
         public override void VisitNamedType(INamedTypeSymbol symbol)
         {
+            if (symbol.DeclaredAccessibility != Accessibility.Protected && symbol.DeclaredAccessibility != Accessibility.Public && symbol.DeclaredAccessibility != Accessibility.NotApplicable)
```

Comment:
We can skip internal and private. The value `NotApplicable` is for assembly and namespace, so it needs to be considered.